packages/cli/src/cli/index.ts

```diff
@@ -418,6 +429,14 @@ const main = async () => {
       await generate(options);
     });
 
+    program
+      .command("login")
```

Comment:
probably should also add a `logout` command that clears any previously saved keys

Reply:
Added a logout command